runatlantis.io/docs/policy-checking.md

```diff
@@ -190,4 +194,34 @@ When the policy check workflow runs, a file is created in the working directory
   }
 ]
 
-```
\ No newline at end of file
+```
+
+## Running policy check only on some repositories
```

Comment:
If i understand the changes correctly, the configuration could be denied explicitly globally and enabled on desired repos too, right?

Reply:
correct, in order to not break backwards compatibility you have to enable it globally (via `--enable-policy-checks`) and exclude the repositories that you want